serverlessworkflow/modules/ROOT/pages/eventing/working-with-openapi-callbacks.adoc

```diff
@@ -0,0 +1,121 @@
+= OpenAPI Callback in {context}
+
+:compat-mode!:
+// Metadata:
+:description: Working with openapi callbacks
+:keywords: kogito, workflow, serverless, openapi, callback, event
+
+This document describes the link:{open_api_spec_url}#callback-object[Openapi Callback] and related examples. Also, it covers the difference between OpenAPI Callback and the Serverless Workflow Callback state. Both the OpenAPI callback and the Callback state perform an action and wait for an event to be produced as a result of the action to resume the workflow. Therefore, both OpenAPI callback and the Callback state are suitable to perform  _fire & wait-for-result_ operations. 
+
+The OpenAPI callback is an asynchronous, out-of-band request that your service will send to some other service in response to specific events. In contrast, the action performed by a Callback state is an asynchronous external service invocation. Both ways aim to send an event to the workflow to resume the flow.
+
+As mentioned in the link:{open_api_spec_url}#callback-object[OpenAPI Callback document], when the workflow makes an asynchronous service call, the callback URL must be provided in the request body. The external service invokes the callback URL and sends a link:{cloud_events_url}[CloudEvent] to the workflow.
```

Comment:
Suggestion to avoid passive voice

```suggestion
As mentioned in the link:{open_api_spec_url}#callback-object[OpenAPI Callback document], when the workflow makes an asynchronous service call, you must provide the callback URL in the request body. The external service invokes the callback URL and sends a link:{cloud_events_url}[CloudEvent] to the workflow.
```